Rakefile

```diff
@@ -41,3 +41,33 @@ namespace :docs do
     system "yardoc -o #{out} --title mongo-#{Mongo::VERSION}"
   end
 end
+
+require_relative "profile/feather_weight_benchmark.rb"
+require_relative "profile/light_weight_benchmark.rb"
+require_relative "profile/middle_weight_benchmark.rb"
+require_relative "profile/heavy_weight_benchmark.rb"
+
+namespace :benchmark do
+
+  task :featherweight do
+    p "FEATHERWEIGHT BENCHMARK"
+    featherweight_benchmark!
+  end
+
+  task :lightweight do
+    p "LIGHTWEIGHT BENCHMARK"
+    lightweight_benchmark!
+  end
+
+  task :middleweight do
+    p "MIDDLEWEIGHT BENCHMARK"
+    middleweight_benchmark!
+  end
+
+  task :heavyweight do
+    p "HEAVYWEIGHT BENCHMARK"
+    heavyweight_benchmark!
+  end
+
+  task :run_all_benchmarks => [:featherweight, :lightweight, :middleweight, :heavyweight]
```

Comment:
you can just call this :all because it's already namedspaced to :benchmark.
So you would run it like:

rake benchmark:all